app/uk/gov/hmrc/bindingtariffclassification/todelete/CaseData.scala

```diff
@@ -24,12 +24,28 @@ import uk.gov.hmrc.bindingtariffclassification.utils.RandomGenerator
 object CaseData {
 
   def createBTIApplication: BTIApplication = {
```

Comment:
Could we name this something like `createBasicBTIApplication` or something along those lines so it's clear we're working with the bare-bones one

Reply:
Updated name to '''createBasicBTIApplication'''